<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-G2TLUC4" name="NX512J" tests="1" failures="0" timestamp="21 Mar 2017 11:09:36 GMT" time="0.02" errors="0">
  <testcase name="@BeforeClass beforeClass" time="0.0" classname="com.zzy.appium.test.HangMei">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at java.io.Reader.<init>(Unknown Source)
	at java.io.InputStreamReader.<init>(Unknown Source)
	at com.zzy.appium.utils.ProUtil.readProperties(ProUtil.java:40)
	at com.zzy.appium.utils.ProUtil.<init>(ProUtil.java:24)
	at com.zzy.appium.test.HangMei.beforeClass(HangMei.java:34)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass beforeClass -->
  <testcase name="@AfterClass afterClass" time="0.0" classname="com.zzy.appium.test.HangMei">
    <skipped/>
  </testcase> <!-- @AfterClass afterClass -->
  <testcase name="f" time="0.0" classname="com.zzy.appium.test.HangMei">
    <skipped/>
  </testcase> <!-- f -->
</testsuite> <!-- NX512J -->
